src/components/QuestionList.js

```diff
@@ -158,26 +158,37 @@ class QuestionList extends React.Component {
       if (this.props.queue.questions.length > 0) {
         questions = this.props.queue.questions.map(questionId => {
           const question = this.props.questions[questionId]
-          return (
-            <Question
-              key={questionId}
-              isUserCourseStaff={this.props.isUserCourseStaff}
-              isUserActiveStaffForQueue={this.props.isUserActiveStaffForQueue}
-              isUserAnsweringQuestion={
-                this.props.userId === question.answeredById
-              }
-              isUserAnsweringOtherQuestion={
-                this.props.isUserAnsweringQuestionForQueue
-              }
-              didUserAskQuestion={this.props.userId === question.askedById}
-              deleteQuestion={() => this.deleteQuestion(questionId)}
-              cancelQuestion={() => this.cancelQuestion(questionId)}
-              startQuestion={() => this.startQuestion(questionId)}
-              finishedAnswering={() => this.handleFinishedAnswering(questionId)}
-              editQuestion={() => this.handleEditQuestion(questionId)}
-              {...question}
-            />
-          )
+          if (
```

Comment:
We could probably use object destructuring to shorten this a bit:

```js
const { queue, userId } = this.props
if (!queue.isConfidential ...) {}
```

Can also take advantage or short circuiting and get rid of the `queue.isConfidential && ` in the second clause, if my boolean logic is working right 😛